tests/org.eclipse.fordiac.ide.test.library/src/org/eclipse/fordiac/ide/test/library/MockDownloader.java

```diff
@@ -77,14 +83,18 @@ public Path downloadLibrary(final String symbolicName, final VersionRange range,
 		final Bundle bundle = Platform.getBundle("org.eclipse.fordiac.ide.test.library"); //$NON-NLS-1$
 
 		try {
-			return Paths.get(FileLocator
-					.toFileURL(FileLocator.find(bundle,
-							new org.eclipse.core.runtime.Path(String.format(FORMAT_STRING, symbolicName, version))))
-					.toURI());
+			return new DownloadResult<>(
+					Paths.get(
+							FileLocator
+									.toFileURL(
+											FileLocator.find(bundle,
+													new org.eclipse.core.runtime.Path(
+															String.format(FORMAT_STRING, symbolicName, version))))
+									.toURI()));
 		} catch (URISyntaxException | IOException e) {
 			// empty
 		}
-		return null;
+		return new DownloadResult<>(DownloadResult.Status.ERROR, "Error"); //$NON-NLS-1$
```

Comment:
This should use the same externalized strings as above

Reply:
externalised the string